mmtrack/models/backbones/mixformer_backbone.py

```diff
@@ -134,9 +158,14 @@ def forward_conv(self, x, t_h, t_w, s_h, s_w):
             search, 'b (h w) c -> b c h w', h=s_h, w=s_w).contiguous()
 
         if self.conv_proj_q is not None:
-            t_q = self.conv_proj_q(template)
-            ot_q = self.conv_proj_q(online_template)
-            s_q = self.conv_proj_q(search)
+            t_q = rearrange(
```

Comment:
For all if-else-None snippets here, We could perform `rearrange` after if-else-None expression since it's general.
It's also the reason of removing `rearrange` from `_build_projection`.